--- babeltrace2-2.1.0/src/bindings/python/bt2/Makefile.am.orig	2025-03-20 06:06:29.014543831 +0100
+++ babeltrace2-2.1.0/src/bindings/python/bt2/Makefile.am	2025-03-20 06:19:16.793365747 +0100
@@ -180,7 +180,7 @@ export SETUPTOOLS_USE_DISTUTILS=stdlib
 endif
 
 install-exec-local: build-python-bindings.stamp
-	$(pyinstall_verbose)opts="--prefix=$(prefix) --exec-prefix=$(exec_prefix) --record $(INSTALLED_FILES) --verbose --no-compile $(DISTSETUPOPTS)"; \
+	$(pyinstall_verbose)opts="--prefix=$(prefix) --exec-prefix=$(exec_prefix) --record $(INSTALLED_FILES) --verbose --optimize=2 $(DISTSETUPOPTS)"; \
 	if [ "$(DESTDIR)" != "" ]; then \
 		opts="$$opts --root=$(DESTDIR)"; \
 	fi; \
